### DISCUSSION TITLE
How to make keda rotate HPAs to new API version ?

### DISCUSSION THREAD

**Discussion by hajdukda** [kedacore/keda#4231] [Q&A / Need Help] [answered]
Hello, we have recently upgraded to keda 2.9.3 and after upgrade I noticed that our HPAs still use "autoscaling/v2beta2" instead of "autoscaling/v2"

How can I force KEDA to replace those old HPA using old api version and replace them with new HPA using v2 ?

Is it as simple as removing old HPA ?

**Comment by zroubalik** [comment, marked as answer]
How did you notice that it is still using "autoscaling/v2beta2"?

**Reply by hajdukda** [reply]
K nvm seems its completely my bad. Read a warning when checking "k describe hpa" and it listed me:
"Warning: autoscaling/v2beta2 HorizontalPodAutoscaler is deprecated in v1.23+, unavailable in v1.26+; use autoscaling/v2 HorizontalPodAutoscaler"
But either way HPA is v2. sorry for confusion.
